migrations/5510e0688fa2_add_llmchallenge_tables.py

```diff
@@ -49,6 +48,16 @@ def upgrade(op=None):
         sa.ForeignKeyConstraint(["user_id"], ["users.id"]),
         sa.PrimaryKeyConstraint("id"),
     )
+    op.create_table(
```

Comment:
Its good to see migrations. Will a contest ever be migrated though? It feels like a new db would be started each time an event is run meaning no migration would need to take place?

Cant hurt to have it, just asking

Reply:
This is where the tables are created, the `llm_model` file just reads the established tables.